packages/manager/modules/iam/src/components/createResourceGroup/createResourceGroup.controller.js

```diff
@@ -110,6 +110,37 @@ export default class CreateResourceGroupController {
     return this.resourceGroup ? 'edit' : 'create';
   }
 
+  /**
+   * A contextualized Set where each key map to a tag
+   * @returns {Object<string,string>}
+   */
+  get tag() {
+    const { mode } = this;
+    if (mode === 'create') {
```

Comment:
suggestion : Maybe you can put a global object (Map) in constants file just to have to call `CONSTANTS_OBJECT[mode]`